src/workerd/api/crypto.h

```diff
@@ -229,7 +250,7 @@ class CryptoKey: public jsg::Object {
       KeyAlgorithm, AesKeyAlgorithm, HmacKeyAlgorithm, RsaKeyAlgorithm,
       EllipticKeyAlgorithm, ArbitraryKeyAlgorithm>;
```

Comment:
ArbitraryKeyAlgorithm isn't really used at the moment – do we still need to have it exported/included in AlgorithmVariant? Do we expect it to have future uses?

Reply:
GitHub doesn't let me select the right line but the hash field in ArbitraryKeyAlgorithm should be `jsg::Optional<kj::OneOf<KeyAlgorithm, kj::StringPtr>` right?

Reply:
Similarly, uint32_t or uint64_t should work better for saltLength in `AsymmetricKeyDetails`, there's no need for salt sizes beyond uint32_t.

Reply:
`uint32_t` is probably good.